events/reporter.go

```diff
@@ -83,7 +83,11 @@ func ReportNewActivation(activation *types.ActivationTx) {
 	if reporter != nil {
 		if err := reporter.activationEmitter.Emit(activationTxEvent); err != nil {
 			// TODO(nkryuchkov): consider returning an error and log outside the function
-			log.With().Error("Failed to emit activation", activation.ID(), activation.PublishEpoch, log.Err(err))
+			log.With().Error("Failed to emit activation",
```

Comment:
I agree, I didn't want to make this PR too big. The global logger probably has to wait until the reporter has been refactored to not be global any more (otherwise how would it log an error? 😉) I didn't want to make this PR too big, but I agree that this should be one of the next steps.

Reply:
> The global logger probably has to wait until the reporter has been refactored to not be global any more (otherwise how would it log an error? 😉)

There is even  a TODO for this :D
```
// TODO(nkryuchkov): consider returning an error and log outside the function
```